concepts/05 UI Components/Form/01 Getting Started with Form/40 Organize the Form Items/30 In Tabs.md

```diff
@@ -0,0 +1,252 @@
+The Form uses the [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) component to display tabs. You can specify the tab panel's settings in the [tabPanelOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#tabPanelOptions) object. A tab can contain any item type.
+
+The following example shows a tabbed item nested in the `Personal Information` group. The resulting Form looks like this:
+
+![DevExtreme Form: Tabbed Items](/images/UiWidgets/form-getting-started-tabs.png)
```

Comment:
There's a strange artifact on this image:
![image](https://user-images.githubusercontent.com/18551316/105339337-9d567d80-5bed-11eb-876c-067c5d86a406.png)